sphinx/src/keywords.rst

```diff
@@ -0,0 +1,231 @@
+***********************
+Restricted Keywords
+***********************
+
+The following Keywords are **restricted** in Version |JSQLPARSER_VERSION| and must not be used for **Naming Objects**: 
```

Comment:
Why are we having references to JSQL here? I thought JSQL would simply use this package

Reply:
Thanks for feedback.

This particular reference is only a variable name used in a template. It will be replaced/filled with the actual version.

Before changing it, we will want to agree on a proper Brand Name for this software and the decision is all yours. Will it be `Presto SQL Parser`, since it is linked to Presto?

Or should it aim broader, like `JSQL:2016', 'JavaSQL16', 'ParseSQL16', 'JSQLSuite', 'JSQLTools'?

Since we are on it, the Git Repo checks out as `sql` (which immediately collided with my  curated collection of the worst SQL statements I have seen in my life.)

Please make a choice and I would adjust everything accordingly.

Reply:
Hmm this is a tricky one. Definitely let's not call it jsql especially since we have C++ version (which your changes will hopefully not impact). Presto SQL parser might be most appropriate

Reply:
So be it! I would adjust tomorrow, remove any manticore too references and submit.
Thanks for the feedback!

I did not touch any C++  Version.

Although: I would like to have a comprehensive Test/CI Suite, which can warn us. If it succeeds, we should be good. When not, then reject.